content/en/docs/Tasks/uninstall-operator.md

```diff
@@ -37,11 +50,15 @@ You can delete the `ClusterServiceVersion` in the namespace that the operator wa
 $ kubectl delete clusterserviceversion <csv-name> -n <namespace>
 ```
 
-By deleting `ClusterServiceVersion`, it will delete the operator's resources that OLM created for the operator such as deployment, pod(s), RBAC, and others. However, `CustomResourceDefinition` (CRD), that is installed by the operator, won't get deleted during uninstalling process. Deleting `CRD` will resolve into deleting all instances of that `CRD` which may be used by other operators. `CRD` can be deleted manually if needed. Furthermore, deleting a `CSV` also deletes any corresponding CSVs that OLM "Copied" into other namespaces watched by the operator.
+### Combine steps 2 and 3
 
 Alternatively, you can delete both `Subscription` and its `CSV` using a sequence of commands:
 ```bash
 $ CSV=kubectl delete subscription <subscription-name> -n <namespace> -o json | jq '.status.installedCSV'
 $ kubectl delete subscription <subscription-name> -n <namespace>
 $ kubectl delete csv $CSV -n <namespace>
-```
\ No newline at end of file
+```
+
+## Step 4: Deciding whetehr or not to delete the CRDs and APIServices
+
+The final step consists of deciding whether or not to delete the CRDs and APIServices that were introduced to the cluster by the operator. Assuming you have already deleted all unwanted Resource on cluster as directed in Step 1, if no resources remain it is safe to remove the CRD or APISerivces. Otherwise, you should not delete the type as the wanted resources will be deleted automatically when the CRD or APISerivce is deleted.
```

Comment:
```suggestion
The final step consists of deciding whether or not to delete the CRDs and APIServices that were introduced to the cluster by the operator. Assuming you have already deleted all unwanted resources on cluster as enumerated in Step 1, if no resources remain it is safe to remove the CRD or APISerivces. Otherwise, you should not delete the type as the wanted resources will be deleted automatically when the CRD or APISerivce is deleted.
```